vscode/package.json

```diff
@@ -249,10 +249,13 @@
     ]
   },
   "scripts": {
-    "tsc:check": "node ../node_modules/typescript/bin/tsc -p ./tsconfig.json",
     "build": "npm run tsc:check && node build.mjs",
     "build:watch": "node build.mjs --watch",
-    "run:web": "npx @vscode/test-web --extensionDevelopmentPath . ../samples"
+    "run:web": "npm exec --no @vscode/test-web --extensionDevelopmentPath . ../samples",
+    "start": "npm exec --no @vscode/test-web --extensionDevelopmentPath . ../samples",
```

Comment:
If this is identical to the line above, we should probably deduplicate and have just one.

Reply:
Any attachment to the naming of `run:web`? I liked `start` because it's idiomatic but I agree having both seems silly